security-policies/bundle/compliance/cis_gcp/rules/cis_1_17/rule.rego

```diff
@@ -9,10 +9,7 @@ default has_cusomter_encrypted_key := false
 finding := result if {
 	data_adapter.is_dataproc_cluster
 
-	result := common.generate_result_without_expected(
-		common.calculate_result(has_cusomter_encrypted_key),
-		data_adapter.resource,
-	)
+	result := common.generate_evaluation_result(common.calculate_result(has_cusomter_encrypted_key))
 }
```

Comment:
this is the common theme in this PR - replace `generate_result_without_expected` with `generate_evaluation_result` which results in sending an object with just `evaluation` field instead of a one including `evidence` too